Extensions/Effects/god-ray-pixi-filter.js

```diff
@@ -0,0 +1,15 @@
+gdjs.PixiFiltersTools.registerFilterCreator('Godray', {
+  makePIXIFilter: function() {
+    var center = new PIXI.Point(0, 0);
+    var godray = new PIXI.filters.GodrayFilter({center});
+
+    return godray;
+  },
+  updateParameter: function(filter, parameterName, value) {
+    if (!['light', 'time', 'gain', 'lacunarity', 'parallel', 'angle', 'x', 'y'].includes(parameterName)) return;
+
+    if (['x', 'y'].includes(parameterName)) filter.center[parameterName] = gdjs.PixiFiltersTools.clampValue(value, -1000, 1000);
+    else if (filter !== 'parallel') filter[parameterName] = gdjs.PixiFiltersTools.clampValue(value, 0, 20);
```

Comment:
nitpicking: associate the block of the if condition with what is in the condition.

Here, we read:

```
if (something related to x/y) doSomethingWithXorY
else if (something with parallel) doSomethingNotRelatedToParallel
else doSomethingWithParallel
```

See the inconsistency?
Keep the less precise case for the `else`:

```
else if (filter === 'parallel') ...
else filter[parameterName] =
```